### PR TITLE
chore: extra lint rules for unit tests - OKTA-260043

### PR DESCRIPTION
**Pull request by aarongranick-okta**
- "fdescribe" or "fit" will cause lint error
- "xit" will cause lint warning
- fixed a couple of tests which were disabled

### CODE REVIEW

File: test/unit/spec/MfaVerify_spec.js [hunk 1]
```diff
@@ -3421,6 +3421,7 @@ function (Okta,
           });
 
           // Do this when we have implemented push errors in OktaAuth and have an example
+          // eslint-disable-next-line jasmine/no-disabled-tests
```
Comment (nikhilvenkatraman-okta) [inline test/unit/spec/MfaVerify_spec.js]: We can probably get rid of this disabled test. Looks like this was added 4 years ago, along with the comment? https://github.com/okta/okta-signin-widget/blame/7074127a62c87f8e59030d26ef5c732594074c9d/test/unit/spec/MfaVerify_spec.js#L3423

File: test/unit/spec/PrimaryAuth_spec.js [hunk 2]
```diff
@@ -1256,7 +1256,7 @@ function (Q, OktaAuth, LoginUtil, Okta, Util, AuthContainer, PrimaryAuthForm, Be
               expect(spyCalls.mostRecent().args).toEqual([BEACON_LOADING_CLS, false]);
             });
         });
-        itp('does not show beacon-loading animation when primaryAuth fails', function () {
+        it('does not show beacon-loading animation when primaryAuth fails', function () {
```
Comment (nikhilvenkatraman-okta) [inline test/unit/spec/PrimaryAuth_spec.js]: why are we updating this ?

Comment (aarongranick-okta) [reply test/unit/spec/PrimaryAuth_spec.js]: I don't remember. I'll change it back and see if it affects anything.

**Comment by nikhilvenkatraman-okta** [comment]
Can you double check the test failure you are seeing in travis ?

**Comment by aarongranick-okta** [comment]
> Can you double check the test failure you are seeing in travis ?

I ran the tests several times. There is a flakiness in `MFAVerify_spec` involving the "Windows Hello" tests. I think we have seen these flake out before with other PRs. There are still `tick()` statements in this test suite, I have a hunch that may be the cause of the flakiness. If I had to guess, I'd say maybe this one: https://github.com/okta/okta-signin-widget/blob/master/test/unit/spec/MfaVerify_spec.js#L556

Since this issue is not related to the changes in the PR, I am going to continue with merge.

**Comment by aarongranick-okta** [comment]
Task for flakey MFA test: https://oktainc.atlassian.net/browse/OKTA-265413

**Pull request merged**